projects/fsastorefrontlib/src/cms-components/checkout/guards/add-options-guard.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import {
+  ActivatedRoute,
+  ActivatedRouteSnapshot,
+  CanActivate,
+  RouterStateSnapshot,
+  UrlTree,
+} from '@angular/router';
+import { Observable } from 'rxjs';
+import { FSCartService } from '../../../core';
+import { map } from 'rxjs/operators';
+import {
+  MultiCartService,
+  OCC_USER_ID_ANONYMOUS,
+  UserIdService,
+} from '@spartacus/core';
+import { Cart } from '@spartacus/core';
+@Injectable({
+  providedIn: 'root',
+})
+export class AddOptionsGuard implements CanActivate {
+  constructor(
+    protected fsCartService: FSCartService,
+    protected activatedRoute: ActivatedRoute,
```

Comment:
activeRoute is now unused and will be removed in the next commit with added test